### DISCUSSION TITLE
No metadata for specific web radio channel

### DISCUSSION THREAD

**Discussion by frank2604** [music-assistant/support#3333] [Q&A] [answered]
I have added my favorit WebRadio channel "Chilltax" (https://www.chilltrax.com) to MA:

http://streams.electronicmusicradiogroup.org:9050/

In a browser and in the MusicCast-App the metadata (artist - title) is shown, in MA not.

Is there anything I can do?

**Comment by OzGav** [comment, marked as answer]
Yeah just tried it myself and nothing. It must be sent in a format not understood by MA.
